webservices/partition/sched_a.py

```diff
@@ -40,6 +42,7 @@ def column_factory(cls, parent):
                 parent.c.contbr_id,
                 parent.c.cmte_id,
             ).label('clean_contbr_id'),
+            sa.cast(sa.func.get_transaction_year(parent.c[cls.transaction_date_column], parent.c.rpt_yr), sa.Numeric(4, 0)).label('transaction_two_year_period'),
```

Comment:
Come to think of it, why do we want a year to be `numeric` in the first place? Wouldn't `integer` or `smallint` work just as well, and avoid the need for the `cast`s?